lib/product-import.coffee

```diff
@@ -210,8 +210,11 @@ class ProductImport
       when 200 then @_summary.updated++
 
   _createProductFetchBySkuQueryPredicate: (skus) ->
-    skuString = "sku in (#{skus.map((val) -> JSON.stringify(val))})"
-    return "masterVariant(#{skuString}) or variants(#{skuString})"
+    if skus.length is 0
+      'masterVariant(sku in ("")) or variants(sku in (""))'
```

Comment:
It is valid query now but produces unnecessary API call. If `skus` is empty than this function should not be called at all and check belongs rather [here](https://github.com/sphereio/sphere-product-import/blob/4884caebe9389b708a6754a515052c0d69377a39/lib/product-import.coffee#L128). Also we should consider logging it as an error as whole matching of product/price import is based on mandatory SKU.